sdk/developersigning/Azure.Developer.Signing/tests/DevSigningClientTests.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Developer.Signing.Tests
+{
+    public class DevSigningClientTests : RecordedTestBase<DevSigningClientTestEnvironment>
```

Comment:
There does not seem to be a test resources file to accompany this; we'll need one of those so that tests can be recorded and run in Live mode.

Reply:
This looks to be the only thing left that is blocking.   See:
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md

Reply:
Thanks, I was running into some challenges with the configuration, but finally just pushed the change on the repo with the test-resources.bicep